src/librustdoc/html/render.rs

```diff
@@ -1307,9 +1369,11 @@ impl Context {
             // Things which don't have names (like impls) don't get special
             // pages dedicated to them.
             _ if item.name.is_some() => {
-                let dst = self.dst.join(&item_path(&item));
-                let dst = try!(File::create(&dst));
-                render(dst, self, &item, true)
+                let _dst = self.dst.join(&item_path(&item));
```

Comment:
Variables starting with a leading underscore are typically ignored, but if this is used it should have a different name